volsupervisor/loop.go

```diff
@@ -16,6 +19,33 @@ var (
 	volumeMutex = &sync.Mutex{}
 )
 
+func (dc *DaemonConfig) updateVolumes() {
+	myVolumes, err := dc.Config.ListAllVolumes()
+	fmt.Println(myVolumes)
+	if err == nil {
```

Comment:
We generally seem to avoid else clauses elsewhere in the codebase when possible.  Can we just return from a `err != nil` and then unindent the rest of the function?